.github/workflows/release.yml

```
@@ -0,0 +1,79 @@
+        run: pnpm install --frozen-lockfile
+      
+      - name: Extract version from tag
+        id: tag_version
+        run: echo "VERSION=${GITHUB_REF#refs/tags/v}" >> $GITHUB_OUTPUT
+      
+      - name: Generate changelog for release
+        run: pnpm changelog:generate --tag v${{ steps.tag_version.outputs.VERSION }}
+        continue-on-error: true  # Don't fail if changelog generation fails
+      
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)